### DISCUSSION TITLE
Speedtest data retention?

### DISCUSSION THREAD

**Discussion by jameszy780** [alexjustesen/speedtest-tracker#1435] [Q&A] [answered]
Hello!

Quick question, how long does speedtest-tracker keep data? is the a variable to change it? I'd like to keep my data for a year or longer. Also it would be handy if there was another drop down that would allow you to filter your results between specific dates.

Thanks!

**Comment by alexjustesen** [comment, marked as answer]
Data is kept until you delete it, or you can use the prune option in General Settings.

**Reply by jameszy780** [reply]
Thank you.
